.buildkite/pipelines/pull_request/base.yml

```diff
@@ -110,8 +111,43 @@ steps:
         - exit_status: '*'
           limit: 1
     artifact_paths:
-      - "target/kibana-security-serverless/**/*"
+      - "target/kibana-security-solution/**/*"
+    
+  - command: .buildkite/scripts/steps/functional/security_serverless_investigations.sh
+    label: 'Serverless Security Investigations Cypress Tests'
+    agents:
+      queue: n2-4-spot
+    depends_on: build
+    timeout_in_minutes: 40
+    parallelism: 4
```

Comment:
Curious - why is this one less? For most of our tests right now, tests will be tagged for both right? Since there's not too much divergence.